src/App.tsx

```diff
@@ -1,10 +1,84 @@
 import React from 'react';
-import Routes from './routes';
+import { Container, AppBar, Toolbar, IconButton, Button, makeStyles } from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import { withRouter } from 'react-router';
 
+import Routes from './routes';
 import './App.css';
+import { auth } from './firebase';
+import logo from './assets/logo.png';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+      flexGrow: 1,
+    },
+    menuButton: {
+      marginRight: theme.spacing(2),
+    },
+    title: {
+      flexGrow: 1,
+    },
+    appBar: {
+        background: '#2c465f',
+    },
+    logo: {
+        borderRadius: 4
+    },
+    toolbar: {
+        justifyContent: 'space-between',
+    },
+    appName: {
+        fontSize: '0.9rem',
+        fontWeight: 500,
+        letterSpacing: '0.02857em',
+    }
+}));
+
+function App(props: any) {
+    const classes = useStyles();
+
+    const signOut = () => {
+        auth.signOut();
+        localStorage.removeItem('user');
+        props.history.push('/');
+    };
+
+    const loggedIn = () => {
+        const userString = localStorage.getItem('user');
+        const user = JSON.parse(userString ? userString : '""');
+
+        if (user) return true;
+        else return false;
+    }
 
-function App() {
-    return <Routes />;
+    return (
+        <>
+            {loggedIn() &&
+                <AppBar position="static" className={classes.appBar}>
+                    <Toolbar className={classes.toolbar}>
+                        <div style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
I think this style should be on the useStyles const

Reply:
Yeah! Of course. Thanks